frontend/src/json_server/db.json

```diff
@@ -0,0 +1,106 @@
+{
+  "staff": [
+    {
+      "id": 1,
+      "first_name": "Joe",
+      "surname": "Bloggs",
+      "start_date": "2018-08-03",
+      "contracted_hours": 40,
+      "hourly_rate": 1.23,
+      "role": "USER",
+      "job_title": "barista",
+      "inactive": false
+    },
+    {
+      "id": 2,
+      "first_name": "Mary",
+      "surname": "McGee",
+      "start_date": "2017-08-12",
+      "contracted_hours": 40,
+      "hourly_rate": 999.99,
+      "role": "MANAGER",
+      "job_title": "supervisor",
+      "inactive": false
+    },
+    {
+      "id": 3,
+      "first_name": "Json",
+      "surname": "Server",
+      "start_date": "2015-01-12",
+      "contracted_hours": 40,
+      "hourly_rate": 0.01,
+      "role": "USER",
+      "job_title": "floor",
+      "inactive": false
+    },
+    {
+      "id": 4,
+      "first_name": "Arthur",
+      "surname": "Guinness",
+      "start_date": "2019-12-12",
+      "contracted_hours": 40,
+      "hourly_rate": 2.99,
+      "role": "USER",
+      "job_title": "chef",
+      "inactive": false
+    }
+  ],
+  "engagement": [
+    {
+      "start": "2018-08-03 00:00",
+      "end": "2018-08-03 01:00",
+      "type": "SHIFT",
+      "hours_worked": 1,
+      "staff_id": 1
+    },
+    {
+      "start": "2018-08-04 09:00",
+      "end": "2018-08-04 17:00",
+      "type": "HOLIDAY",
+      "hours_worked": 10,
+      "staff_id": 1
+    },
+    {
+      "start": "2017-08-03 10:00",
+      "end": "2017-08-03 20:00",
+      "type": "SHIFT",
+      "hours_worked": 10,
+      "staff_id": 2
+    },
+    {
+      "start": "2018-08-03 00:00",
+      "end": "2018-08-13 00:00",
+      "type": "HOLIDAY",
+      "hours_worked": 80,
+      "staff_id": 2
+    },
+    {
+      "start": "2018-08-03 00:00",
+      "end": "2018-08-03 01:00",
+      "type": "SHIFT",
+      "hours_worked": 1,
+      "staff_id": 3
+    },
+    {
+      "start": "2018-08-03 00:00",
+      "end": "2018-08-03 01:00",
+      "type": "HOLIDAY",
+      "hours_worked": 1,
+      "staff_id": 3
+    },
+    {
+      "start": "2018-08-05 12:00",
+      "end": "2018-08-05 17:00",
+      "type": "SHIFT",
+      "hours_worked": 3,
+      "staff_id": 4
+    },
+    {
+      "start": "2020-08-03 00:00",
+      "end": "2021-08-03 01:00",
```

Comment:
Is this such a long holiday on purpose?

Reply:
no, not on purpose. ( I just copied the data from the backend, data.sql file).
The idea is that as we define the responses expected form the endpoints, we will update the data in this file.